tests/format/vue/vue/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1594,6 +2123,25 @@ export default {
 ================================================================================
 `;
 
+exports[`case-sensitive-tags.vue - {"bracketSameLine":true} format 1`] = `
+====================================options=====================================
+bracketSameLine: true
+parsers: ["vue"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<template>
+	<Input></Input>
+</template>
+
+=====================================output=====================================
+<template>
+  <Input></Input>
+</template>
+
+================================================================================
```

Comment:
This file is hard to review, can you add a new test?

Reply:
created new tests